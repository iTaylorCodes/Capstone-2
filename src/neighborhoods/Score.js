/** Score.
 *
 * Component dedicated to showing a neighborhood score.
 *
 * Renders a score provided by the walkscore api.
 *
 * Rendered by Scores.
 */
const Score = ({ score }) => {
  return (
    <div
      className="Score m-3 rounded-circle p-2"
      style={{ width: "125px", height: "125px", background: "#5DD95D" }}
    >
      <h2>{score.score}</h2>
      <p>{score.description}</p>
    </div>
  );
};

export default Score;
