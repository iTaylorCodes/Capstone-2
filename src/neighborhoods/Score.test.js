import Score from "./Score";
import { render } from "@testing-library/react";

const score = { score: 10, description: "score" };

it("renders without crashing", () => {
  render(<Score score={score} />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Score score={score} />);
  expect(asFragment()).toMatchSnapshot();
});
