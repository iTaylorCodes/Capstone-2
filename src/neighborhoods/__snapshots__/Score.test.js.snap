// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Score bg-info m-3 rounded-circle p-2"
    style="width: 125px; height: 125px;"
  >
    <h2>
      10
    </h2>
    <p>
      score
    </p>
  </div>
</DocumentFragment>
`;
